share/wake/lib/system/remote_cache_runner.wake

```diff
@@ -0,0 +1,357 @@
+# Copyright 2024 SiFive, Inc.
```

Comment:
Some of this file (but not most of it) was already reviewed. `git` failed to notice the rename from `remote_cache.wake` -> `remote_cache_runner.wake`